aliasMultinomial.lua

```diff
@@ -0,0 +1,13 @@
+local AM = torch.class('torch.AliasMultinomial') 
```

Comment:
This has to preprocess though. So, it will still not be a single call like `torch.multinomial` but a setup and then `torch.aliasMultinomialDraw(n)`